state.tf

```diff
@@ -2,15 +2,34 @@ provider "azurerm" {
   features {}
 }
 
+provider "azurerm" {
+  alias           = "aks-infra"
+  subscription_id = "${var.aks_subscription_id}"
```

Comment:
applies throughout, string interpolation not needed if it's just a variable, it will be causing lots of warnings like this
```suggestion
  subscription_id = var.aks_subscription_id
```

Reply:
yep 84 of them